### PR TITLE
Update contributing guidelines

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Adds _a lot_ of detail to the contributing guidelines.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -65,3 +230,12 @@ bin/appropriate-images.js industryRealEstateBanner
 ```
 
 *Note: You'll see that the generated images in `src/img/dist/` have filenames corresponding to the sizes you've defined.*
+
+### Videos
+
+Guidelines for videos:
+
+- Must be in the `src/video/` folder.
+- Must be in MP4 format with `type: mp4` specified in the example's front-matter.
```
Comment (langsmith) [inline CONTRIBUTING.md]: > in the example's front-matter

What does this mean?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -65,3 +230,12 @@ bin/appropriate-images.js industryRealEstateBanner
 ```
 
 *Note: You'll see that the generated images in `src/img/dist/` have filenames corresponding to the sizes you've defined.*
+
+### Videos
+
+Guidelines for videos:
```
Comment (langsmith) [inline CONTRIBUTING.md]: Any guidelines regarding video dimension and/or file size?

Comment (colleenmcginnis) [reply CONTRIBUTING.md]: Video dimension will be determined by the device used, right? Right now we only have a device frame for Pixel so ideally videos would be shot on a Pixel to provide the correct dimensions. 

No guidelines related to file size at the moment. If that changes, I'll update here and make updates to existing videos.

File: CONTRIBUTING.md (hunk 1 above)
Comment (colleenmcginnis) [reply CONTRIBUTING.md]: Oh that's a copy-paste error. Should just be: 

Must be in MP4 format ~~with `type: mp4` specified in the example's front-matter~~.

I'll make that update.

**Comment by langsmith** [comment]
I approved this @colleenmcginnis

**Comment by colleenmcginnis** [comment]
Shoot sorry I forgot about this @langsmith! Merging now 😄

**Pull request merged**